### PR TITLE
New power breaker with alarm threshold

### PR DESCRIPTION
**Pull request by kirovilya**
TONGOU model TO-Q-SY2-163JZT https://aliexpress.ru/item/1005005444617697.html

waiting https://github.com/Koenkk/zigbee-herdsman/pull/701

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -1175,13 +1288,15 @@ module.exports = [
...
+            {modelID: 'TS0202', manufacturerName: '_TZ3040_wqmtjsyk'},
         ],
         model: 'TS0202',
         vendor: 'TuYa',
         description: 'Motion sensor',
         whiteLabel: [{vendor: 'Mercator Ikuü', model: 'SMA02P'},
             {vendor: 'TuYa', model: 'TY-ZPR06'},
-            {vendor: 'Tesla Smart', model: 'TS0202'}],
+            {vendor: 'Tesla Smart', model: 'TS0202'},
+            {vendor: 'MiBoxer', model: 'PIR1-ZB'}],
```
Comment (Koenkk) [inline devices/tuya.js:1299]: Please use new whitelabel here (such that device will be detected correctly): https://github.com/Koenkk/zigbee-herdsman-converters/blob/e630d74e36761c806a5e05f321570d2bb486bee9/devices/tuya.js#L1029

Comment (kirovilya) [reply devices/tuya.js]: done

**Comment by Koenkk** [comment]
thanks!

**Comment by kirovilya** [comment]
thanks, I also thought how to extend the "extend"-object with additional attributes :)

**Comment by henkiejan1** [comment]
@Koenkk @kirovilya  Very nice that this device should be added. But i didn't see it in the changelog of the Zigbee2mqtt 1.30.4 that's just released? :)

**Comment by Koenkk** [comment]
@henkiejan1 it is (`TS011F_with_threshold`)

**Comment by henkiejan1** [comment]
@Koenkk i see and works like a charme! Only need to delete and add again to got the whole functions work! Thank you very much! :tada:

**Pull request merged**